opba-protocols/opba-protocol-api/src/main/java/de/adorsys/opba/protocol/api/errors/ProcessErrorStrings.java

```diff
@@ -0,0 +1,10 @@
+package de.adorsys.opba.protocol.api.errors;
+
+public final class ProcessErrorStrings {
```

Comment:
We use `@UtilityClass` for this - i.e. `de.adorsys.opba.api.security.external.domain.HttpHeaders`